Apps/FlightSoftware/fprime/CubeRover/WatchDogInterface/WatchDogInterface.hpp

```diff
@@ -17,55 +17,69 @@
 #define WatchDogInterface_HPP
 
 #include "CubeRover/WatchDogInterface/WatchDogInterfaceComponentAc.hpp"
+#include <CubeRover/WatchDogInterface/WatchDogRxTask.hpp>
 #include "Include/FswPacket.hpp"
 
 #include "lin.h"
 #include "adc.h"
 #include "sci.h"
 
 // The pin number for the deploment pin 2
-const U8 deploy_bit = 4;
+static const U8 deploy_bit = 4;
 // The number of thermistors on the SBC
-const U8 number_thermistors = 6;
+static const U8 number_thermistors = 6;
 // Default size of zero sent to watchdog
-const U16 zero_size = 0x0000;
+static const U16 zero_size = 0x0000;
 // Minimum size that should be received back from watchdog when receiving
-const U8 min_receive_size = 8;
+static const U8 min_receive_size = 8;
 // Magic Value first sent in the header between watchdog and hercules communication
-const U32 header_magic = 0x21B00B;
+static const U32 header_magic = 0x21B00B;
+
+// This controls how long we'll wait for a response before allowing the next message of the same type
+// to be sent to the MSP430 watchdog
+static const uint32_t COMMAND_TIMEOUT_MILLISECONDS = 2000;
```

Comment:
Is this also an arbitrarily picked value?

Reply:
Somewhat arbitrarily, though this is more based on what I feel is reasonable. The other ones above this that I commented on about being arbitrary were picked nearly out of thin air, while this one I selected as something that seemed reasonable. Put another way, if we call those other ones guesses then this is closer to a hypothesis -- not at all proven, but based on some amount of reason and past experience.